client/plots/sampleScatter.interactivity.js

```diff
@@ -103,7 +103,7 @@ export function setInteractivity(self) {
 					const shape = showColor ? self.getShape(chart, d, 1, true) : self.getShape(chart, d)
 					const width = value.length * 9 + 60
 					const svg = td.append('svg').attr('width', width).attr('height', '35px')
-					const g = svg.append('g').attr('transform', 'translate(10, 16)')
+					const g = svg.append('g').attr('transform', 'translate(10, 18)')
```

Comment:
thanks it works to make the size constant in tooltip. but it's still small and hard to see. can u make it the same size as showing in scatter legend

Reply:
Fixed now